src/test/java/com/browserstack/examples/config/WebDriverConfiguration.java

```diff
@@ -0,0 +1,103 @@
+package com.browserstack.examples.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.List;
+
+public class WebDriverConfiguration {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(WebDriverConfiguration.class);
```

Comment:
The Logger object has not been used in the class

Reply:
@sanketsmali03 https://github.com/browserstack/browserstack-examples-junit5/pull/7#issuecomment-889748455 is resolved.